packages/ui/molecules/Accordion/Accordion.twig

```diff
@@ -0,0 +1,22 @@
+<div data-component="Accordion">
```

Comment:
**issue/non-blocking**: Add comment for component api and blocks

Reply:
There are other files missing this part, I would like to close this PR in order for `develop` to have the latest documentation.